### DISCUSSION TITLE
Best Accuracy for Object Detection (Persons) with Nvidia GPU?

### DISCUSSION THREAD

**Discussion by AnshulJ999** [codeproject/CodeProject.AI-Server#231] [Q&A]
Hi, 

So my set up is: Personal Laptop with Nvidia 3050ti GPU and Windows 10 is running the Code Project AI server, 2.9.4, and using the model Yolov5.6.2. I've tried Yolov8 too. 

Home Assistant is running on another laptop, which is my home server, and it has this Code Project integration running: https://github.com/codeproject/CodeProject.AI-HomeAssist-ObjectDetect (it hasn't been updated in a while though it seems to work for now)

I'm using a Tapo C212 camera (currently set to 1080p resolution). Not sure if increasing res to 2k will have much of an effect on accuracy?

So everything is kinda working (bit unreliable at times), but my main issue is the accuracy is a bit low. It detects a person is there even when I'm not there, maybe because there are some posters on the wall behind. And sometimes it does not detect me even when i am there. So both false positives and false negatives. 

What can I do to improve accuracy? Or rather, which model would be the most accurate? 

I know training a model is possible but im new to all this so that'd be a bit tough. So out of the presets; which one would be most accurate? And which model size too? 

Thanks.

**Comment by ChrisMaunder** [comment]
Then larger the model, the better the accuracy (in general). Also, the more specific a model, the better the accuracy.

Have you tried Mike Lud's custom models? His IPCam-General is tuned for typical webcam scenarios and so may prove to be a better fit.

**Reply by AnshulJ999** [reply]
Thanks so much for the quick reply. I have just heard about these custom models for the first time. https://github.com/MikeLud/CodeProject.AI-Custom-IPcam-Models

These are the correct ones? Is there any guide on installing these? Kinda new to this. 

If I understood correctly I need to install this file: YOLOv8 Models/Custom Models/ipcam-general-v8.pt 

EDIT: Okay there are many models marked 'ipcam-general' in the Repo, and different file formats, so I am quite confused actually. 

Appreciate any help, thanks!

**Reply by MikeLud** [reply]
YOLOv8 module comes with my custom models you do not need to download the from my GitHub repo.

**Comment by ChrisMaunder** [comment]
@mikelud can answer better regarding that set of custom models. 

The IPCam-general model is already included in the installation of the YOLOv5 (Python and .NET) and YOLOv8 modules
